### PR TITLE
Adds landing page with project description

### PR DESCRIPTION
**Pull request by jaysaleh**
This pr adds a landing page for the project.

Screenshots: https://screenshot.googleplex.com/6gSHPy8F2Oh.png

### CODE REVIEW

File: src/main/webapp/index.html [hunk 1]
```diff
@@ -3,6 +3,55 @@
...
+      annotations can include comments, questions, and replies. Students watching can easily refer to relevant comments
+      made by other students, add their own, and reply to other students’ questions.
+    </p>
+    <br />
+    <p>
+      ZoomTube was developed by Amy (<a href="https://github.com/amuamushu" class="link" target="_blank">@amuamushu</a
+      >), Jay (<a href="https://github.com/jaysaleh" class="link" target="_blank">@jaysaleh</a>), and Nathan (<a
+        href="https://github.com/n-wach"
+        class="link"
+        target="_blank"
```
Comment (n-wach) [inline src/main/webapp/index.html:39]: Inconsistent formatting here, can you clean this up?

File: src/main/webapp/style.css [hunk 2]
```diff
@@ -11,13 +18,17 @@ html {
   position: relative;
 }
 
-#add-btn-cont {
+#add-btn-cont,
+#view-btn-cont {
+  align-items: center !important;
   margin: 30px auto auto auto;
```
Comment (amuamushu) [inline src/main/webapp/style.css:24]: I think you forgot to alphabetize the declarations in this selector.

File: src/main/webapp/style.css [hunk 3]
```diff
@@ -101,15 +115,24 @@ html {
...
   margin: auto;
   width: 50%;
 }
 
-@font-face {
-  font-family: 'Product Sans';
-  font-style: normal;
-  font-weight: 400;
-  src: url('./fonts/product-sans/ProductSansRegular.ttf') format('truetype');
+#view-btn-cont a {
```
Comment (amuamushu) [inline src/main/webapp/style.css:130]: Can you group the selectors and add section comments? I think it would be helpful if we're only using 1 style sheet.

https://google.github.io/styleguide/htmlcssguide.html#Section_Comments

File: src/main/webapp/style.css (hunk 2 above)
Comment (jaysaleh) [reply src/main/webapp/style.css]: fixed

File: src/main/webapp/style.css (hunk 3 above)
Comment (jaysaleh) [reply src/main/webapp/style.css:130]: Waiting for Cassandra and Fola

File: src/main/webapp/index.html [hunk 4]
```diff
@@ -3,6 +3,55 @@
...
+  crossorigin="anonymous"
+/>
 
-<a href="/list.html">View list of lectures</a>
+<body>
+  <nav class="navbar navbar-light">
+    <h2 id="header-text">ZoomTube</h2>
+  </nav>
+  <div id="text">
+    <h1 class="display-6">Project Overview</h1>
```
Comment (f-camp) [inline src/main/webapp/index.html:18]: Do you think we can replace this with "What is Zoomtube?"

WDYT?

Comment (jaysaleh) [reply src/main/webapp/index.html:18]: I don't have a preference. @amuamushu @n-wach what do you think?

File: src/main/webapp/index.html [hunk 5]
```diff
@@ -3,6 +3,55 @@
...
+    <br />
+    <p>
+      ZoomTube is a web application that facilitates the viewing of recorded university lectures. With classes shifting
+      towards a virtual environment, lectures have become disengaging, especially for those watching asynchsronously.
+      Our web application aims to mitigate this issue by providing a way for students to interact with each other on a
+      single platform.
+    </p>
+    <br />
+    <p>
+      Recorded university lectures are synchronized with their transcript and a student discussion forum. Within this
```
Comment (billie-cat) [inline src/main/webapp/index.html:28]: I feel these 2 sentences are a repetition of the first. The first one mentions the student discussion forum, so I am not sure what these sentences add in terms of information.

File: src/main/webapp/style.css [hunk 6]
```diff
@@ -1,3 +1,10 @@
+@font-face {
```
Comment (billie-cat) [inline src/main/webapp/style.css:1]: Before we keep adding to this style.css and make it really big, lets split this up. Please do this in a separate PR.

File: src/main/webapp/style.css (hunk 3 above)
Comment (billie-cat) [reply src/main/webapp/style.css:130]: We do not need it alphabetized, but can you please split this css into multiple files? You can pause this PR and get this fix out by branching off of master. Then get that approved and merged and then just sync this branch.

File: src/main/webapp/index.html (hunk 4 above)
Comment (n-wach) [reply src/main/webapp/index.html:18]: sure let's try "What is ZoomTube?"

**Comment by n-wach** [comment]
Please merge with master to use the new directory structure and split CSS files

File: src/main/webapp/index.html (hunk 5 above)
Comment (jaysaleh) [reply src/main/webapp/index.html]: I think both of these sentences are crucial, the first explains the high level synchronization feature while the next goes into how students would use this product.

File: src/main/webapp/lectures/lecture-list.css [hunk 7]
```diff
@@ -72,3 +69,12 @@
   margin: auto;
   width: 70%;
 }
+
+#submit-btn-cont {
+  margin: auto;
+  width: 150px;
+}
```
Comment (n-wach) [inline src/main/webapp/lectures/lecture-list.css:76]: This rule is already included above

File: src/main/webapp/lectures/lecture-list.css [hunk 8]
```diff
@@ -72,3 +69,12 @@
...
+
+#submit-btn-cont .btn {
+  background-color: #4c8bf5 !important;
+}
```
Comment (n-wach) [inline src/main/webapp/lectures/lecture-list.css:80]: This one too

File: src/main/webapp/index.html [hunk 9]
```diff
@@ -3,6 +3,56 @@
...
+      >), Jay (<a href="https://github.com/jaysaleh" class="link" target="_blank">@jaysaleh</a>), and Nathan (<a
+        href="https://github.com/n-wach"
+        class="link"
+        target="_blank"
+        >@n-wach</a
+      >) as a capstone project for Google's 2020 STEP internship program.
+    </p>
+  </div>
+  <div id="view-btn-cont">
+    <a class="btn" href="/list.html">View list of lectures</a>
```
Comment (n-wach) [inline src/main/webapp/index.html:46]: Fix link
